evm/vm/forks/sharding/vmc_handler.py

```diff
@@ -158,6 +168,9 @@ def __init__(self, *args, default_privkey, **kwargs):
 
         super().__init__(*args, **kwargs)
 
+    def get_default_sender_address(self):
```

Comment:
This function doesn't appear to be used anywhere.

Reply:
Actually I used them in `test_vmc_handler.py`, to get the `default_sender_address` for doing transactions.
Should I remove them? I can replace them by pass the address in use as argument in those testing functions in `test_vmc_handler.py`

Reply:
I must have missed it.  disregard.